utils/perl5lib/Config/SetupTools.pm

```diff
@@ -218,180 +218,6 @@ sub getxmlvars
     }
 }
 
-#-------------------------------------------------------------------------------
```

Comment:
This set_compiler perl function doesn't appear to be used anymore. I didn't want to try to rewrite it to accept a different location for the .cime directory, so I'm just removing this unused code.